src/App.js

```diff
@@ -1,25 +1,46 @@
-import logo from './logo.svg';
-import './App.css';
+import { Fragment, useState, useEffect } from "react";
+import { getUser, getFolder } from "./api/api";
+import "./global.css";
+import Layout from "./components/Layout/Layout";
+import FolderInfo from "./components/FolderInfo/FolderInfo";
+import SharePage from "./pages/SharePage/SharePage";
+import SearchBar from "./components/SearchBar/SearchBar";
+import CardList from "./components/CardList/CardList";
+import OnlyCard from "./components/OnlyCard/OnlyCard";
 
 function App() {
+  const [userProfile, setUserProfile] = useState(null);
+  const [profile, setProfile] = useState(null);
+  const [folderName, setFolderName] = useState(null);
```

Comment:
P2: initial value를 넘기지 않으면 undefined 인데 null로 넘겨야 할 이유가 잇을까요?
```suggestion
  const [userProfile, setUserProfile] = useState();
  const [profile, setProfile] = useState();
  const [folderName, setFolderName] = useState();
```